index.js

```diff
@@ -0,0 +1,41 @@
+var slideIndex = 1;
+slides(slideIndex);
+
+// Next/previous controls
+function plusSlides(n) {
+  slides((slideIndex += n));
+}
+
+// Thumbnail image controls
+function currentSlide(n) {
+  slides((slideIndex = n));
+}
+
+function slides(n) {
+  var i;
+  var slides = document.getElementsByClassName("slides");
+  var dots = document.getElementsByClassName("dot");
+  if (n > slides.length) {
```

Comment:
Using const or let would be better here.

Reply:
Okay will make the changes